src/trackingModules/canvas/index.js

```diff
@@ -0,0 +1,7 @@
+import popupMenuTracking from './popupMenu';
+
+export default {
+  __depends__: [
+    popupMenuTracking
+  ]
+};
```

Comment:
Not sure if we need 2-level structure: `trackingModules/canvas/popupMenu`.

Reply:
The idea is that we could group events like contextpad/palette/popup menu separately from diagram modeling events (such as moving, updating elements). So we would have `diagram.*` and `canvas.*` events for example. 

I think this allows also for filtering for a certain type of events based on the prefix when analysing the data. But we can decide to discuss this and consider if it actually brings value.

Reply:
Perhaps we may want to use some kind of filtering, but I don't see a containment relation between canvas and popupMenu. It's rather something like `ui.{popupMenu, contextPad, palette}`.

Reply:
Still it's a naming discussion which we can as well postpone until we are about to use the module.

Reply:
We can, but we can also try to abstract categories once we have a couple of events :)

Reply:
I switched this to "controls" following [this conversation](https://camunda.slack.com/archives/GP70M0J6M/p1681456224817459).

I'm still going to have a conversation about event naming with Christian but since it doesn't affect the implementation, I think we can go ahead with this and naming can be changed later if need be.